pkg/runs/runsGet.go

```diff
@@ -26,9 +26,17 @@ var (
 	// Make a map of how many hours for each unit so can compare from and to values consistently
 	// Can be extended to support other units
 	timeUnits = map[string]int{
-		"w": 168,
-		"d": 24,
-		"h": 1,
+		"w": 10080,
+		"d": 1440,
+		"h": 60,
+		"m": 1,
+	}
+
+	timeUnitsForErrorMap = map[string]string{
+		"w": "weeks",
+		"d": "days",
```

Comment:
Hmm. Two maps holding data about a time unit ? Shouldn't the time unit be a class/enum with values and getters ?